internal/ignition/ignition.go

```diff
@@ -234,7 +238,15 @@ func (g *installerGenerator) Generate(ctx context.Context, installConfig []byte)
 		g.log.Errorf("Failed to write file %s", installConfigPath)
 		return err
 	}
-
+	// TODO: remove this once installer support bootstrap-in-place https://github.com/openshift/installer/pull/4482
+	if g.cluster.HighAvailabilityMode != nil && *g.cluster.HighAvailabilityMode == models.ClusterHighAvailabilityModeNone {
+		// In case of single node rename bootstrap-in-place-for-live-iso.ign to bootstrap.ign
+		err = os.Rename(filepath.Join(g.workDir, "bootstrap-in-place-for-live-iso.ign"), filepath.Join(g.workDir, "bootstrap.ign"))
```

Comment:
So why do we use other name in the beginning?
I think we should either assume installer to generate it with the new name and use as-is (so we have clarity of the scenario by looking at the file name)
Or, using ``bootstrap.ign`` everywhere

Reply:
Currently, the installer requires the env and generates the ignition under a different name (bootstrap-in-place-for-live-iso.ign).
I hope later this week, the installer will converge towards (not requiring the env and generating the ignition with the same name) https://github.com/openshift/enhancements/pull/565#discussion_r545545030
Using it "as is" requires more changes (in assisted-installer and assisted-service getClusterFiles API).
So let's see what the openshift installer ends up duing and adjust later.